MimeKit/MimeKit.csproj

```diff
@@ -6,6 +6,7 @@
     <VersionPrefix>3.1.1</VersionPrefix>
     <Authors>Jeffrey Stedfast</Authors>
     <TargetFrameworks>netstandard2.0;netstandard2.1;net452;net46;net47;net48;net5.0;net6.0</TargetFrameworks>
+    <LangVersion>10</LangVersion>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Comment:
I'm guessing this is needed for the `==` -> `is` changes?

Reply:
Yep -- better pattern matching, and better codegen on .NET6.0 for string interpolation. Our min targets hold us back from utilizing much else.